analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -343,12 +343,14 @@ data class GemSpec(
             )
         }
 
+        private val GITHUB_REGEX = Regex("https*:\\/\\/github.com\\/(?<owner>[\\w-]+)\\/(?<repo>[\\w-]+)")
+
         // Gems tend to have GitHub URL set as homepage. Seems like it is the only way to get any VCS information out of
         // gemspec files.
         private fun parseVcs(homepageUrl: String): VcsInfo =
```

Comment:
I don't like having this method here in general. We have similar heuristics in other places, however not necessarily in ORT code (e.g. `go get` has a list of known code hosting sites used to infer VCS type). Maybe move it to `VersionControlSystem` or `VcsInfo` as `inferFromUrl()` or something like that?

Reply:
I agree, and I was also thinking about introducing helper functions like `isGitHubUrl`. But let's do that in a separate PR, as that would be a larger change.